pandas/tests/test_sorting.py

```diff
@@ -216,10 +239,19 @@ def test_int64_overflow_issues(self):
         tm.assert_series_equal(out["left"], result, check_names=False)
         assert result.name is None
 
+    @pytest.mark.slow
+    @pytest.mark.parametrize("how", ["left", "right", "outer", "inner"])
```

Comment:
i think we have a fixture for this somewhere? (fix in followup)

Reply:
After a quick look doesn't appear like we have a fixture in a conftest